deploy/infrastructure/examples/README.md

```diff
@@ -0,0 +1,68 @@
+# DSS Infrastructure Examples
+
+This folder contains deployment examples for various environments:
+- Cloud Provider:
+  - *-google: Google Cloud Engine
+- Deployment size:
+  - mini-*: Low resources for development
+  - prod-*: Production grade resources
+
+## Infrastructure
+
+### Prerequisites
+Download & install the following tools to your workstation:
+
+1. Install [terraform](https://developer.hashicorp.com/terraform/downloads).
+2. Install provider specific tools:
+   1. [Google Cloud Engine](./README.md#google-cloud-engine)
+3. Install tools from [Prerequisites](../../../build/README.md)
+
+#### Google Cloud Engine
+
+1. Install and initialize [Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk).
+   1. Confirm successful installation with `gcloud version`.
+2. Check that the DSS project is correctly selected: gcloud config list project
+    1. Set another one if needed using: `gcloud config set project $GOOGLE_PROJECT_NAME`
+3. Enable the following API using [Google Cloud CLI](https://cloud.google.com/endpoints/docs/openapi/enable-api#gcloud):
+    1. `container.googleapis.com`
+    2. If you want to manage DNS entries with terraform: `dns.googleapis.com`
+4. Install the auth plugin to connect to kubernetes: `gcloud components install gke-gcloud-auth-plugin`
+
+### Deployment of the Kubernetes cluster
+
+1. Copy or edit in place an example folder to `/deploy/infrastructure/personal/`. (Note that the modules can be added to existing projects) 
+2. Edit `terraform.tfvars` and set the variables according to your environment.
+3. Initialize terraform: `terraform init`.
+4. Run `terraform plan` to check that the configuration is valid. It will display the resources which will be provisioned.
```

Comment:
If it's the user's first time on google cloud (like me), `gcloud auth application-default login` should also be run